doc/testing_samples/samples_list.rst

```diff
@@ -1,14 +1,66 @@
-.. _samples_list:
-
-List of Sidewalk application samples
-####################################
-
-.. toctree::
-   :maxdepth: 1
-   :glob:
-   :caption: Subpages:
-
-   sidewalk_application_samples/sensor_monitoring.rst
-   sidewalk_application_samples/template_subghz.rst
-   sidewalk_application_samples/template_ble.rst
-   sidewalk_application_samples/dut.rst
+.. _samples_list:
+
+Samples overview
+################
+
+.. note::
+    For the best user experience, start your work with the Sensor Monitoring application.
+
+In the Sidewalk repository, all samples are placed in the :file:`samples` directory.
+
+The following table demonstrates configuration for the supported development kits and differences between samples:
+
+.. tabs::
+
+   .. tab:: nRF52840
+
+      +-----------------------------+--------------------------------------------------------------------------+-----------------------------------------------+-------------------------------+-----------------------+--------------+-------------+-------------+
+      | Sample                      | Additional hardware                                                      | Bootloader                                    | Application overlays          | Bluetooth® LE support | LoRa support | FSK support | DFU support |
+      +=============================+==========================================================================+===============================================+===============================+=======================+==============+=============+=============+
+      | :ref:`sensor_monitoring`    | * `Semtech SX1262MB2CAS`_ (optional)                                     | --                                            | * :file:`fsk.conf`            | Supported             | Supported    | Supported   | --          |
+      |                             |                                                                          |                                               | * :file:`lora.conf`           |                       |              |             |             |
+      |                             |                                                                          |                                               |                               |                       |              |             |             |
+      +-----------------------------+--------------------------------------------------------------------------+-----------------------------------------------+-------------------------------+-----------------------+--------------+-------------+-------------+
+      | :ref:`template_subghz`      | * External flash (included in the DK)                                    | * MCUboot - DFU partition in external flash   | * :file:`fsk.conf`            | --                    | Supported    | Supported   | Supported   |

```

Comment:
I have a doubt here: In sub-Ghz template BLE is not supported as a Sidewalk transport layer. However it is used for DFU and (in LoRa transport) for registration.
But maybe adding too much details in an overview table isn't good solution. idk